### ISSUE TITLE
Release for linux-aarch64

### ISSUE THREAD

**Issue by jamilraichouni** [redhat-developer/vscode-xml#930]
Hi!

Would love to see a Linux arm64/ aarch64 build for lemminx. 
Is that somewhat on the agenda? 😀

Many thanks!
Jamil

**Comment by angelozerr** [comment]
@rgrunber @datho7561 do you think it is easily doable?

**Comment by rgrunber** [comment]
I assume this is referring to the lemminx binary, since it should be possible to install the generic vscode-xml extension on linux-aarch64. For the binary, the main issue would be that GitHub doesn't provide such runners : https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources .

**Comment by angelozerr** [comment]
@rgrunber I think this issue can be closed?

**Comment by angelozerr** [comment]
Ok thanks @datho7561 for your clarification!

**Comment by jay-babu** [comment]
> Depends on [actions/runner-images#5631](https://github.com/actions/runner-images/issues/5631)

This has been closed as of 4 days ago. Is it possible now? @rgrunber @datho7561

**Comment by rgrunber** [comment]
Yes (and GraalVM supports it), but based on https://github.blog/changelog/2024-06-03-actions-arm-based-linux-and-windows-runners-are-now-in-public-beta/ they are not freely available for public repositories.

**Comment by javalsai** [comment]
Despite not being supported natively by runners, it's easily doable with virtualization using an action such as https://github.com/uraimo/run-on-arch-action. It's used by several other projects to provide binaries for alternative architectures.
